server/src/test/java/org/elasticsearch/cluster/coordination/NodeJoinExecutorTests.java

```diff
@@ -100,7 +100,7 @@ public void testPreventJoinClusterWithUnsupportedIndices() {
         Settings.builder().build();
         Metadata.Builder metaBuilder = Metadata.builder();
         IndexMetadata indexMetadata = IndexMetadata.builder("test")
-            .settings(settings(Version.fromString("6.8.0"))) // latest V6 released version
+            .settings(settings(IndexVersion.fromId(6080099))) // latest V6 released version
```

Comment:
Nit/question: using underscores like in IndexVersion could improve readability/searchabilty? `6_08_00_99`

Reply:
We havent done that in constants elsewhere - eg `Queries.NESTED_DOCS_IDENTIFIED_VIA_PRIMARY_TERMS_VERSION`. These hardcoded numbers tend to be write-once-read-never anyway...